### PR TITLE
Bugfix/ios footer titles not clickable

### PR DESCRIPTION
**Pull request by AndyOGo**
improves #213

### CODE REVIEW

File: src/components/m-dropdown/mixins/dropdown.scss [hunk 1]
```diff
@@ -148,6 +148,9 @@
 @mixin dropdown-toggle() {
   position: relative;
 
+  // this is important to enable click events on iOS
+  cursor: pointer;
```
Comment (LucaMele) [inline src/components/m-dropdown/mixins/dropdown.scss]: mmmmm maybe better separate this inanother mixing (single responsability)

Comment (AndyOGo) [reply src/components/m-dropdown/mixins/dropdown.scss]: well, every toggle button needs it anyway. Cause even at desktop the expected style is `pointer` on hover.

Comment (AndyOGo) [reply src/components/m-dropdown/mixins/dropdown.scss]: or do mean to extract those 2 lines into a mixin and include it here?

**Comment by AndyOGo** [comment]
@LucaMele 
Sorry we need this asap, i will merge now

**Pull request merged**